src/providers/map.service.ts

```diff
@@ -9,13 +9,25 @@ export class MapService {
   options = { timeout: 5000, enableHighAccuracy: true };
   markers = [];
   windows = [];
+  gmaps_key;
   constructor() {}
 
-  downloadGoogleMaps(cb: Function) {
+  downloadGoogleMaps(cb: Function, public platform: Platform) {
     window['mapsCb'] = cb;
     let head = document.getElementsByTagName('head')[0];
     let mapsApi = document.createElement('script');
-    mapsApi.src = `https://maps.googleapis.com/maps/api/js?libraries=places,geometry&key=${ENV.gmaps_key}&callback=mapsCb`;
+    if(this.platform.is('cordova') && (this.platform.is('core')){
+      if(ENV === 'prod'){
+        gmaps_key = ENV.gmaps_core;
+      } else {
+        gmaps_key = ENV.gmaps_dev;
+      }
+    } else {
+      // will need separate gmaps vars for android and ios
```

Comment:
I think we ought to do this now. I don't think it makes sense to merge/deploy a key that doesn't have origin restrictions.

Reply:
OK. I was holding off because @sherson requested to get the google maps website API key up ASAP. I didn't document that anywhere, my fault.

Reply:
@dfaulken I was having trouble figuring out the android API restrictions, so we took this approach for now. I'm investigating the android part right now.